docs/hub/datasets-image.md

```diff
@@ -0,0 +1,123 @@
+
+```
+images/1.jpg
+images/2.png
+images/3.tiff
+images/4.webp
+```
+
+## Additional columns
+
+If there is additional information you'd like to include about your dataset, like text captions or bounding boxes, add it as a `metadata.csv` file in your repository. This lets you quickly create datasets for different computer vision tasks like text captioning or object detection.
+
+```
+train/1.jpg
+train/2.jpg
+train/3.jpg
+train/4.jpg
+train/metadata.csv
+```
+
+Your `metadata.csv` file must have a `file_name` column which links image files with their metadata:
+
+```csv
+file_name,text
+1.jpg,a drawing of a green pokemon with red eyes
+2.jpg,a green and yellow toy with a red nose
+3.jpg,a red and white ball with an angry look on its face
+4.jpg,a cartoon ball with a smile on it's face
+```
+
+You can also use a JSONL file `metadata.jsonl`:
+
+```jsonl
+{"file_name": "1.jpg","text": "a drawing of a green pokemon with red eyes"}
+{"file_name": "2.jpg","text": "a green and yellow toy with a red nose"}
+{"file_name": "3.jpg","text": "a red and white ball with an angry look on its face"}
+{"file_name": "4.jpg","text": "a cartoon ball with a smile on it's face"}
+```
+
+Note that all the files must be in a directory that gives the split name (e.g. `train` or `test`, see [File names and splits](./datasets-file-names-and-splits) for more information).
+
+## Relative paths
+
+You can also locate the images in a different location than the metadata file:
+
+```
+train/images/1.jpg
+train/images/2.jpg
+train/images/3.jpg
+train/images/4.jpg
+train/metadata.csv
+```
+
+To access the image, the `file_name` column should be a full relative path to the file, not just the filename:
+
+```csv
+file_name,text
+images/1.jpg,a drawing of a green pokemon with red eyes
+images/2.jpg,a green and yellow toy with a red nose
+images/3.jpg,a red and white ball with an angry look on its face
+images/4.jpg,a cartoon ball with a smile on it's face
+```
+
+## Image classification
+
+For image classification datasets, you can also use a simple setup: use directories to name the image classes. Store your image files in a directory structure like:
+
+```
+green/1.jpg
+green/2.jpg
+
+red/3.jpg
+red/4.jpg
+```
+
+The dataset created with this structure contains two columns: `image` and `label` (with values `green` and `red`).
+
+You can also use provide multiple splits. To do so, your dataset directory should have the following structure (see [File names and splits](./datasets-file-names-and-splits) for more information):
+
+```
+train/green/1.jpg
+train/red/3.jpg
+
+test/green/2.jpg
+test/red/4.jpg
+```
+
+## Parquet format
+
+Instead of uploading the images as files, along with a metadata file for the other columns, you can also embed them inside a [Parquet](https://parquet.apache.org/) file. This case is useful if you have a large number of images, if you want to embed multiple image columns, or if you want to store additional information about the images in the same file.
```

Comment:
```suggestion
Instead of uploading the images and metadata as individual files, you can also embed everything inside a [Parquet](https://parquet.apache.org/) file. This is useful if you have a large number of images, if you want to embed multiple image columns, or if you want to store additional information about the images in the same file.
```